src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("The excavator has just started working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("The excavator was stopped");

```

Comment:
Similarly, the message in the 'stopWork' method should indicate the type of machine, 'Excavator', that has stopped working to improve the clarity of the message.